packages/lit-dev-content/site/tutorials/content/custom-attribute-converter/01.md

```diff
@@ -77,4 +61,6 @@ export class MyElement extends LitElement {
 
 {% endswitchable-sample %}
```

Comment:
@e111077  In lieu of a better place to put this comment, splatting it here:

 I really don't want to have any code showing the following:
```ts
  @property()
  dateStr = this.date.toString();
```
The part I'm concerned about is a one-time initialization of a property based on the initial value of another property, even if it's labeled as "not best practice" It's not clear from the text that this is _also_ bad because `dateStr` would never re-evaluate `this.date.toString()` once `this.date` changes. 

Can we just leave out the initializer for `dateStr`? It doesn't seem like it's actually needed to make the point? That would fix my concern.

Reply:
Makes sense can do

Reply:
I've included these changes making it an empty string in #805. Didn't touch the md so that we don't have a merge conflict @arthurevans

Reply:
Ace. Updated.